notary-integration-test/src/integration-test/kotlin/integration/eth/WithdrawalPipelineIntegrationTest.kt

```diff
@@ -46,13 +48,16 @@ class WithdrawalPipelineIntegrationTest {
 
     private val withdrawalService: Job
 
+    /** Relay vacuum config */
+    private val relayVacuumConfig = loadConfigs("relay-vacuum", RelayVacuumConfig::class.java, "/eth/vacuum.properties")
```

Comment:
Please, add `createVacuumConfig` to `ConfigHelper`. See `withdrawalServiceConfig`.